### ISSUE TITLE
Possibility of focusing multiple folders and file.

### ISSUE THREAD

**Issue by icnahom** [symfurkan/rsl-vsc-focused-folder#3]
This is a feature that would be very feasible to have when using the extension. 
It would be amazing if we had the ability to focus on more that one file or folder.

**Comment by langscot** [comment]
This feature would make this extension 2% useful to 100% useful.

**Comment by rcherny** [comment]
Right. Why can't the Focused Folder pane list more than one? That'd be awesome.

**Comment by ItsCubeTime** [comment]
+1 to this.

There should also be a way of focusing folders outside of your workspace!

**Comment by mattalxndr** [comment]
+1

**Comment by UrosVuj** [comment]
> This is a feature that would be very feasible to have when using the extension. It would be amazing if we had the ability to focus on more that one file or folder.

@icnahom  @97lunarmare  @mattalexx  @rcherny @ItsCubeTime 

So, since I needed this functionality and this extension doesn't cover it, plus it hasn't received updates for a year, I made a new one with the ability to select multiple folders (and/or files) and some other features. <br /> [Here it is :)](https://marketplace.visualstudio.com/items?itemName=UrosVujosevic.explorer-manager) <br /> Cheers  :tada:

**Comment by icnahom** [comment]
@UrosVuj It's a very nice extension, love the simplicity!
